lib/resolvers/pil.js

```diff
@@ -3,7 +3,26 @@ const { pick } = require('lodash');
 const { generateLicenceNumber } = require('../utils');
 
 module.exports = ({ models }) => ({ action, data, id }, transaction) => {
-  const { PIL } = models;
+  const { PIL, PilTransfer } = models;
```

Comment:
I should have beefed about `PIL` vs `Pil` in your other PR.

The 💩 is out of the 🐴 now though.

Reply:
I considered both, but `PIL` already causes us grief in some places due to the !camelCase . Also, we've also already got `AsruEstablishment`